apps/token-manager/app/src/components/InfoBoxes.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Box,
+  GU,
+  Text,
+  TokenBadge,
+  breakpoint,
+  textStyle,
+  theme,
+  useTheme,
+} from '@aragon/ui'
+import { useNetwork } from '@aragon/api-react'
+import { formatBalance, stakesPercentages } from '../utils'
+import You from './You'
+import LocalIdentityBadge from './LocalIdentityBadge/LocalIdentityBadge'
+
+const DISTRIBUTION_ITEMS_MAX = 7
+
+const displayedStakes = (accounts, total, colors) => {
```

Comment:
Should this function be memoized?

Reply:
Yes absolutely! (I’ll memoize its result from the component function)